src/libstd/io/mod.rs

```diff
@@ -1332,10 +1332,11 @@ pub trait Seek {
 
     /// Returns the length of this stream (in bytes).
     ///
-    /// This method is implemented using three seek operations. If this method
-    /// returns successfully, the seek position is unchanged (i.e. the position
-    /// before calling this method is the same as afterwards). However, if this
-    /// method returns an error, the seek position is undefined.
+    /// This method is implemented using uo to three seek operations. If this
```

Comment:
uo?

Reply:
*sigh*

It's supposed to be "up". Sorry for that and thanks for catching the typo. Gonna fix in a minute...